client/app/components/Input/InputSwitch.scss

```diff
@@ -0,0 +1,38 @@
+@import "../../styles/_global.scss";
+
+.switch {
+  @include setFontSize($size-20);
+  display: inline-block;
+  min-width: 120px;
+
+  &Wrapper {
+    display: flex;
+    flex-direction: row;
+    border-radius: $size-4;
+    padding: $size-4;
+  }
+
+  &On {
+    @include linearTransition(0.5s);
+    background: $key-lime;
+  }
+
+  &Off {
+    @include linearTransition(0.5s);
+    background: $mulberry-70;
+    justify-content: flex-end;
```

Comment:
Was actually wondering what you think about having the off state have a left-aligned toggle knob, and the on state have a right-aligned toggle knob. 🤔

Was a bit unaccustomed to seeing it in reverse.

Reply:
I can switch it, now that you point it out I think you're correct. https://www.w3schools.com/howto/howto_css_switch.asp also handles it the way you mentioned.